graphql-database-manager-dynamo/src/main/java/com/fleetpin/graphql/database/manager/dynamo/DynamoDb.java

```diff
@@ -99,14 +96,17 @@ public DynamoDb(ObjectMapper mapper, List<String> entityTables, DynamoDbAsyncCli
         if (entity.getCreatedAt() == null) {
             setCreatedAt(entity, Instant.now()); //if missing for what ever reason
         }
+        final int revision = entity.getRevision();
```

Comment:
We could have the table class have an incrementRevision method.

Reply:
I can't hide it from the consumers. Feel should only be visible to the DB layer.

We do need to expose setRevision for usecase when you contain the revision in your mutation to confirm editing latest.